.github/workflows/build_package.yml

```diff
@@ -67,35 +71,56 @@ jobs:
             build-family: macos
             build-package: py-runtime-pkg
             experimental: true
+
+          # Windows packages.
```

Comment:
I can update the section in https://openxla.github.io/iree/reference/bindings/python/ after a nightly + stable release.

![image](https://github.com/openxla/iree/assets/4010439/a8c2790c-c934-479f-958b-7b64b7f5854d)